et_ee.lang

```diff
@@ -237,10 +237,10 @@ gui.xaero_open_slime=Ava klahvivajutusel
 gui.xaero_options=Valikud
 gui.xaero_make_automatic=Tee maailm/server autom.
 gui.xaero_make_automatic_msg1=Kas oled kindel?
-gui.xaero_make_automatic_msg2=See vahetab kõigi valitud maailma/serveri teepunktide andmed automaatse vastu, simuleerides valitud maailma/serveri automaatseks tegemist. Veendu, et tead, mida teed.
+gui.xaero_make_automatic_msg2=See vahetab kõik teepunktide andmed valitud maailmas/serveris automaatse vastu, simuleerides valitud maailma/serveri automaatseks tegemist. Veendu, et tead, mida teed.
```

Comment:
I still have the exact same issue with this.

Reply:
"This swaps all waypoint data in selected world/server with automatic one, thus simulating making the selected world/server automatic."

Reply:
Maybe you should just reword the English one? :D

Reply:
I use the word "between" in the original:
_This will swap all waypoint data between the selected world/server and the auto one._
This makes it more obvious that the transfer goes in both directions. Can you do something similar?
If not, you could also switch the "selected" and the "auto" in the sentence, which should make it less of a problem.

Reply:
Okay, I guess I did not indeed realize it goes both ways. 

But in that case, why highlight the fact that "it simulates making the selected world/server automatic"? Or more specifically, what is the use of "automatic" if it gets manual data now?

Reply:
Thanks, makes more sense now!
The manually selected data A becomes "automatic" because, if the data A is transferred to where the current automatic data B is, then, the next time you join the world, data A will be visible automatically and data B would be something you would have to select manually. Basically, you can imagine a slot for data called "automatic" and if data A is in the slot, then A is automatic.